common/beerocks/scripts/prplmesh_utils.sh.in

```diff
@@ -287,21 +287,46 @@ status_function() {
     pgrep -l ieee1905_transport
     pgrep -l local_bus
 
-    # check for operational status
-    LOGS_PATH=@BEEROCKS_LOG_FILES_PATH@
+    bridge_mac="$(ip link show dev @BEEROCKS_BRIDGE_IFACE@ | awk '/^ *link\/ether / {print $2}')"
+    bml_cmd="bml_get_device_operational_radios $bridge_mac"
```

Comment:
It's used only once, why do you need a variable for that?

Reply:
cleaner code.
When I write it in a single line
It's harder to understand, this way it's clear that the BML command uses the bridge's mac address for it's functionality

Reply:
```suggestion
    bml_cmd="bml_get_device_operational_radios $(ip link show dev @BEEROCKS_BRIDGE_IFACE@ | awk '/^ *link\/ether / {print $2}')
```

This line is harder to understand then using a variable for the bridge's mac

Reply:
Ok